test/functionalTest/cases/0706_direct_https_notifications/direct_https_notifications.test

```diff
@@ -0,0 +1,138 @@
+# Copyright 2017 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Direct HTTP notification (accepting self signed certs)
+
+--SHELL-INIT--
+${SCRIPT_HOME}/httpsPrepare.sh --keyFileName /tmp/harnessTest.key --certFileName /tmp/harnessTest.pem
+
+dbInit CB
+brokerStart CB 0 IPV4 -acceptSelfSignedCert
```

Comment:
It's a little confusing, thinking about the CLI param `-https` that the broker has already.
The way it is now, the broker is started without -https so it does not accept https requests, however, it does accept https notifications ...  
Perhaps we should rethink the CLIs a little ...
I find it somehow confusing.